api/security/__init__.py

```diff
@@ -12,6 +13,7 @@ def __init__(self, app: Scaffold = None, running_local=False):
 
     def init_app(self, app: Scaffold):
         if self.running_local:
+            CORS(app)
```

Comment:
suggestion: I think this refactor affect the tests of both the SecurityHeader extension and the build_flask_app function